src/VisualJsonEditor/Samples/Sample.schema.json

```diff
@@ -17,6 +17,11 @@
       "type": "string",
       "default": "n/a"
     },
+    "StringValueEncodedBase64": {
+      "type": "string",
+      "default": "n/a",
+      "contentEncoding": "base64"

```

Comment:
This seems strange, "contentEncoding" is not really a JSON Schema field. I think we should use "format" "byte" with "type" "string" for that to be compliant with JSON Schema: 

https://github.com/RicoSuter/NJsonSchema/blob/958afa39a8386a6f02bf6604aca742e4d18acb61/src/NJsonSchema/JsonFormatStrings.cs#L62

Reply:
Well, I don't know but it is clearly defined in the specs:
https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.8.3

Alternatively, I couldn't find any mention of the "byte" format:
https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.7.3

Maybe I'm missing something?

Anyway, maybe we should handle both case?

Reply:
Maybe "byte" is OpenAPI/Swagger specific (NJsonSchema is also used in NSwag)

It would also make sense to add "contentEncoding" and the other "new" properties to the NJsonSchema class... can you create an issue there just that we dont forget it?

Reply:
`contentEncoding` seems actually to be followed by an existing issue: https://github.com/RicoSuter/NJsonSchema/issues/574

Reply:
Ah yes, quite some stuff missing :-)

"byte" is defined here: 
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md#data-types

Reply:
Sounds good with me!